src/lib.rs

```diff
@@ -111,13 +124,24 @@ pub struct Images {
     pub gcp: Option<GcpImage>,
 }
 
-/// Convert a string e.g. `stable` or `rhcos-4.8` to a stream URL.
+/// Convert a string e.g. `fcos-stable` or `rhcos-4.8` to a stream URL.
+/// The format is `<distro>-<stream>`.
 pub fn stream_url_from_id(s: impl AsRef<str>) -> Result<String> {
     let s = s.as_ref();
-    Ok(if s.starts_with("rhcos-") {
-        rhcos::StreamID::from_str(s)?.url()
-    } else {
-        fcos::StreamID::from_str(s)?.url()
+    let mut it = s.splitn(2, '-');
```

Comment:
It will be nice when https://doc.rust-lang.org/std/primitive.str.html#method.split_once is stable (and propagated far enough it's in our MSRV).